Arena.toml

```diff
@@ -1,358 +1,216 @@
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (10:25)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (68:66)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W003::[Completeness]::Medium] missing parameter meta within task: files (5:6-5:23)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W003::[Completeness]::Medium] missing parameter meta within task: labels (4:6-4:26)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W003::[Completeness]::Medium] missing parameter meta within task: ref_files (6:6-6:27)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (84:45-84:56)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (85:46-85:65)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (86:26-86:39)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (87:39-87:52)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (91:18-91:27)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "ENCODE-DCC/chip-seq-pipeline2:/dev/test/test_task/compare_md5sum.wdl"
-message = "[v1::W012::[Style, Clarity]::Low] string defined with single quotes (93:17-93:36)"
-
-[[concerns]]
-kind = "LintWarning"
-document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (133:46)"
+commit_hash = "a44b18a30fd6192af713fded4bc824f6e77f6ff9"
+filters = ["/template/task-templates.wdl"]
 
-[[concerns]]
-kind = "LintWarning"
+[[diagnostics]]
 document = "getwilds/ww-fastq-to-cram:/ww-fastq-to-cram.wdl"
-message = "[v1::W001::[Spacing, Style]::Low] trailing space (182:63)"
+message = "/ww-fastq-to-cram.wdl:129:5: note: task `FastqToUnmappedBam` has an extraneous parameter metadata key named `unmapped_bam` [rule: MatchingParameterMeta]"
```

Comment:
@a-frantz What do you think of having the rule at the end of the line? I don't feel strongly enough to suggest we change it, but it feels more natural at the beginning with the file name and location.

Reply:
Lol your suggestion is how @peterhuene had it initially and I asked for a change to the back 😂 Though I am far from tied to this message template and would be happy to revisit.

Reply:
Give me a second to dig up our original thread.

Reply:
https://github.com/stjude-rust-labs/wdl/pull/68#discussion_r1630375699

Reply:
Ah, I figured I had missed something with it now at the end of the line. As I said, I don't feel strongly enough to disagree with the placement.

Reply:
I unresolved your resolve @adthrasher lol. Now that you've pointed it out I'm second guessing. @peterhuene feel free to weigh in.

My initial concern was that `<severity>[<ID>]` wasn't clear that ID was the "hey look this value up in the documentation". You mentioned it feeling "natural" to have the rule name near the front and I think you're correct.

Current:
```
message = "read_group.wdl:32:12: warning: struct member name `ID` is not snake_case [rule: SnakeCase]"
```
New?
```
message = "read_group.wdl:32:12: warning[SnakeCase]: struct member name `ID` is not snake_case"
```

Reply:
I'm good with switching it back as I didn't have an opinion one way or the other originally.

I'll revert that and update the baselines.

Reply:
I like your (@a-frantz) updated format. That feels natural.

Reply:
I've pushed the change.